CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+* Bug fixes
```

Comment:
Do you mind adding the intended new release number here as well? We can update as needed, but you can probably guess the number and url, etc.